pkgs/development/libraries/mesa/default.nix

```
@@ -45,7 +45,7 @@ self = stdenv.mkDerivation {
       "ftp://ftp.freedesktop.org/pub/mesa/${version}/mesa-${version}.tar.xz"
       "ftp://ftp.freedesktop.org/pub/mesa/older-versions/${branch}.x/${version}/mesa-${version}.tar.xz"
     ];
-    sha256 = "0dwyk7cxlwna24ap23i8i92a2qcc6xhp16i03zzakpqiz4i03jxi";
+    sha256 = "09nlyjw6pjn4fr3adxxqnlm56kbas0lqrh022sdnkrxqyh1cicqc";
   };
 
   prePatch = "patchShebangs .";
```

```suggestion
  postPatch = "
    patchShebangs .
  '';
```
otherwise patches might not apply and also we should be more specific what is being patched.